x-pack/plugins/apm/server/routes/services/route.ts

```diff
@@ -263,8 +254,9 @@ const servicesDetailedStatisticsRoute = createApmServerRoute({
       environment,
```

Comment:
Can't we replace it all by `params.query`?

Reply:
I like explicitness. It will help catch keys that are removed for instance.